.circleci/config.yml

```diff
@@ -1,5 +1,36 @@
 version: 2
 jobs:
+  unit-php54-mage18:
+    docker:
+      - image: italoc/php54
+      - image: circleci/mysql:5.7
+    steps:
+      - checkout
+      - run:
+          name: PHP 5.4 Magento 1.8
+          command: |
+            mkdir ./artifacts
+            export TEST_ENV=php54-mage18
+            PHPUNIT_PHAR=tests/unit/phpunit-4.8.36.phar MAGENTO_VERSION=magento-mirror-1.8.1.0 tests/scripts/ci.sh
+      - store_artifacts:
+          path: ./artifacts
+
+  unit-php54-mage19:
```

Comment:
you can de-dupe the two above by making magento version a param

Reply:
@matt-thomason we might just want clean up this whole yml. If you are busy ignore this a d make this a tech debt

Reply:
Nah I'll make it right